src/PropertyManager/Travely.PropertyManager.Service/Models/Commands/AddPropertyCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Travely.PropertyManager.Service.Models.Commands
+{
+    public class AddPropertyCommand
+    {
+        public string Name { get; set; }
+        public byte Stars { get; set; }
+        public string Address { get; set; }
+        public double? Latitude { get; set; }
```

Comment:
You have a doubles here but converting to decimal in time of Model Creating , why ?

Reply:
I couldn't find the place where the double to decimal conversion happens, please provide more details.